docs/topics/proc_configuring-your-application-to-use-vertx.adoc

```diff
@@ -1,16 +1,15 @@
 [id='configuring-your-application-to-use-vertx_{context}']
 = Configuring your application to use {VertX}
 
-To configure your application to use {VertX}:
+Reference the `pom.xml` file in the {VertX} BOM (Bill of Materials) artifact to configure your application.
 
 .Prerequisites
 
 * A Maven-based application
 
 .Procedure
 
-. Reference the {VertX} BOM (Bill of Materials) artifact in the `<dependencyManagement>` section of the `pom.xml` file of your application.
-Specify the `<type>pom</type>` and `<scope>import</scope>`:
+. Add `<type>pom</type>` and `<scope>import</scope>` to the `<dependencyManagement>` section of the `pom.xml` file:
```

Comment:
This is better, but factually incorrect. You need to add the `io.vertx:vertx-dependencies` artifact that isn't there in the first place, and specify the `<type>` and `<scope>` along with it. @spisiakm, correct me if I'm wrong, please.

Reply:
That's right. I would say something like

> Add `vertx-dependencies` dependency along with proper `scope` and `type` to the dependency management section of the pom.xml file.

Reply:
@tradej @spisiakm I addressed your latest commits in my most recent commit. I will point out that the example pom.xml referenced later in the runtime guide does include the dependency management artifact.